src/System.Net.Http/tests/FunctionalTests/HttpRequestMessageTest.cs

```diff
@@ -14,7 +14,7 @@ namespace System.Net.Http.Functional.Tests
 {
     public class HttpRequestMessageTest
     {
-        Version _expectedRequestMessageVersion = new Version(1, 1);
+        Version _expectedRequestMessageVersion = PlatformDetection.IsUap ? new Version(2,0) : new Version(1, 1);
```

Comment:
I personally prefer having two separate tests, one for each platform. That makes the behavior difference clear and visible.
(In other words, test names/attributes should be sufficient as a functional specification of the component.)